src/patternfly/components/DataList/data-list.scss

```diff
@@ -213,13 +213,12 @@
   margin-left: var(--pf-c-data-list__toggle--MarginLeft);
 
   // rotate icon
-  .pf-c-button > * {
+  .pf-c-button:focus > * {
     pointer-events: none;
-    transition: var(--pf-c-data-list__toggle--c-button--Transition);
-    transform: var(--pf-c-data-list__toggle--c-button--Transform);
 
     .pf-c-data-list__item.pf-m-expanded & {
-      --pf-c-data-list__toggle--c-button--Transform: var(--pf-c-data-list__item--m-expanded__toggle--c-button--Transform);
+      transition: var(--pf-c-data-list__toggle-icon--m-expanded--Transition);
```

Comment:
Same here, you'll want to move the `transition` to `.pf-c-button > *`, then just use `transform` for the expanded state. And update the var name so it matches the selectors. Looks like it was in the correct format before.